go/vt/vtgate/planbuilder/operators/subquery_planning.go

```diff
@@ -22,6 +22,8 @@ import (
 
 	"golang.org/x/exp/slices"
 
+	"vitess.io/vitess/go/vt/vtgate/evalengine"
+
```

Comment:
Newline after this seems strange? Should be part of the block after this I think?

Reply:
I could not care less about imports. If `goimports` is happy, I'm happy